app/Http/Requests/StoreWebhook.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreWebhook extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'action'    => 'integer|required',
+            'url'       => 'string|required',
```

Comment:
Please also add https://laravel.com/docs/10.x/validation#rule-active-url, and a max length to avoid mysql errors